Build fails under GCC on Linux x86_64

```
~/code/wren $ gcc --version
gcc (Ubuntu 4.8.2-19ubuntu1) 4.8.2
Copyright (C) 2013 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

~/code/wren $ make
cc -c -std=c99 -Wall -Werror -Os -Iinclude -o build/release/main.o src/main.c
src/main.c: In function ‘runRepl’:
src/main.c:84:12: error: ignoring return value of ‘getline’, declared with attribute warn_unused_result [-Werror=unused-result]
     getline(&line, &size, stdin);
            ^
cc1: all warnings being treated as errors
make: *** [build/release/main.o] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -27,7 +27,7 @@ debug: prep wrend
 
 # Debug command-line interpreter.
 wrend: $(DEBUG_OBJECTS)
-	$(CC) $(CFLAGS) $(DEBUG_CFLAGS) -Iinclude -lm -o wrend $^
+	$(CC) $(CFLAGS) $(DEBUG_CFLAGS) -Iinclude -o wrend $^ -lm
 
 # Debug object files.
 build/debug/%.o: src/%.c include/wren.h $(HEADERS)
@@ -38,7 +38,7 @@ release: prep wren
 
 # Release command-line interpreter.
 wren: $(RELEASE_OBJECTS)
-	$(CC) $(CFLAGS) $(RELEASE_CFLAGS) -Iinclude -lm -o wren $^
+	$(CC) $(CFLAGS) $(RELEASE_CFLAGS) -Iinclude -o wren $^ -lm
 
 # Release object files.
 build/release/%.o: src/%.c include/wren.h $(HEADERS)

--- a/src/main.c
+++ b/src/main.c
@@ -81,10 +81,10 @@ static int runRepl(WrenVM* vm)
 
     char* line = NULL;
     size_t size = 0;
-    getline(&line, &size, stdin);
+    ssize_t num_read = getline(&line, &size, stdin);
 
     // If stdin was closed (usually meaning the user entered Ctrl-D), exit.
-    if (feof(stdin))
+    if (feof(stdin) || num_read == -1)
     {
       printf("\n");
       return 0;

